src/main/java/be/syntra/devshop/DevshopFront/controllers/SearchController.java

```diff
@@ -72,69 +74,68 @@ public String searchProductDescription(@RequestParam String description, Model m
 
     @GetMapping("/sortbyname")
     public String sortByName(Model model) {
+        disablePriceSorting();
+        reverseNameSorting();
```

Comment:
You do these 2 calls in 4 methods.

Reply:
There's 2 times disable**Price**Sorting() + reverseNameSorting() and 2 times disable**Name**Sorting() + reverseNameSorting()
;-)